nomic/project.py

```diff
@@ -665,6 +665,86 @@ def _get_atoms(self, ids: List[str]) -> List[Dict]:
         else:
             raise Exception(response.json())
 
+    def get_tags(self) -> Dict[str, str]:
```

Comment:
perhaps an option for passing a list of datum_ids whose tags you want would be useful? maybe not a blocker for this pr though

Reply:
Will keep just this for now. I want to see how users use it prior to writing different ways of getting tags.